website/src/pages/docs/guides/graphql-server-apollo-yoga-with-server-preset.mdx

```diff
@@ -0,0 +1,349 @@
+import { PackageCmd, Callout, Tabs, Tab } from '@theguild/components'
+
+# Guide: GraphQL Yoga / Apollo Server with Server Preset
```

Comment:
This is a separate guide from https://the-guild.dev/graphql/codegen/docs/guides/graphql-server-apollo-yoga. I want to keep them separate for now, and put a callout in the original article to suggest this article instead?

Alternatively, we could update the original title but does abruptly change the content cause confusion and (maybe) SEO issues? Happy to hear thoughts/advices here.

Reply:
I agree with keeping both in the meantime until we have some more feedback/usage of the server preset. Lets also add a Callout to the other doc page for encouraging people to try this approach!

Reply:
This is done in https://github.com/dotansimha/graphql-code-generator/pull/8858/commits/bda2c1d56d95aa82b35fd64f0baf766b0205cb52 and looks something like this:

![Screen Shot 2023-02-01 at 10 59 39 pm](https://user-images.githubusercontent.com/33769523/216036813-d95d3e85-ea65-4b79-a911-7432d183a757.png)

Happy to make changes to the text or position if needed :)